tests/test_blocks.py

```diff
@@ -245,3 +246,24 @@ def test_logs_with_compression(self):
                 query = 'SELECT 1'
                 client.execute(query, settings=settings)
                 self.assertIn(query, buffer.getvalue())
+
+    def test_logs_message_format(self):
+        log_regex_source = (
+            r'^\[ .+? \] '                 # hostname
+            r'[\d-]{10} [\d:]{8}\.\d{6} '  # timestamp
+            r'\[ \d+? \] '                 # thread_id
+            r'\{[\w-]{36}\} '              # query_id
+            r'<[a-z]+?> '                  # priority
+            r'\w+?:'                       # source
+        )
+        log_regex = re.compile(log_regex_source, re.I)
+
+        with capture_logging(
+            'clickhouse_driver.log', 'INFO', as_queue=True
```

Comment:
The intention was to check each record separately, and avoid potential failures in case of potential multiline log messages.

In this case `buffer.getvalue().splitlines()` will also work -- can switch to this option.